docs/pom.xml

```diff
@@ -125,5 +143,21 @@
 				</plugins>
 			</build>
 		</profile>
+		<profile>
+			<id>spring-cloud-gcp-ci-it</id>
+			<build>
+				<plugins>
+					<plugin>
+						<groupId>org.apache.maven.plugins</groupId>
+						<artifactId>maven-failsafe-plugin</artifactId>
+						<configuration>
+							<systemPropertyVariables>
+								<spring.cloud.gcp.firestore.project-id>spring-cloud-gcp-ci-firestore</spring.cloud.gcp.firestore.project-id>
```

Comment:
Can we set this env var externally in the build config?

Reply:
Per standup convo - moving tests is probably the better choice here.

Reply:
I moved this into the test itself because it wasn't picking it up when I put it into `application-test.properties`

Reply:
So can this be removed?

Reply:
Yes, removed. I originally was trying to find a way to do it outside of `-P spring-cloud-gcp-ci-it` since that profile only exists on `spring-cloud-gcp-samples`. Alas, we do what we have to do...